README.md

```diff
@@ -190,3 +231,24 @@ $ curl http://localhost:8000/0/tx-history
   }
 }
 ```
+
+### Stop a wallet
+
+Stop the wallet and its connections and remove it from the store.
+
+```bash
+$ curl -X POST --data "key=key"  http://localhost:8000/wallet/stop
+{"success":true}
+```
+
+## Util scripts
+
+### Generate new words
+
+Generate a new seed (24 words)
```

Comment:
I'd say: "This script generates a new seed with 24 words. It is useful when you'd like to start a brand new wallet. The seed is a sensitive data that cannot be shared with anyone else. Remember to clean up your clipboard after copying & pasting."

Reply:
Is this absolutely necessary? I don't like that we are incentivizing this kind of API, it's really bad to have this level of sensitive information being passed around like this.

It's one thing to have a vulnerability that allows an attacker to make an unauthorized transaction. It is another to have a vulnerability that leeks a seed that allows a transaction at any point in the future to be made potentially without leaving traces of where de leek happened, and leaving no other fix but to change every seed ever generated while there was a leek.

If this is absolutely necessary in the immediate future, I'd rather not document it and possible add this behind a `/deprecated`, `/unsafe`, `/not-recommended` or something like that.

Reply:
@jansegre this is not an API, just an util script to be run locally.

Reply:
Yes, it just logs to the user screen and nothing else. It's just a helper to generate a new seed, so users don't need to use other ways (wallet desktop, wallet mobile, or executing our method in the lib).

Reply:
Oh, my bad. OK then.